_data/meltano/extractors/tap-recruitee/rawwar.yml

```diff
@@ -25,40 +25,18 @@ settings:
   kind: string
   label: API URL
   name: api_url
-  value: https://api.mysample.com
+  value: https://api.recruitee.com
 - description: The token to authenticate against the API service
   kind: password
   label: Auth Token
   name: auth_token
-- description: "'True' to enable schema flattening and automatically expand nested\
```

Comment:
@rawwar this is great, thanks for the update!

Would you mind leaving these default setting in the definition file: flattening_enabled, flattening_max_depth, stream_map_config, stream_maps? They come with the SDK and usually makes it easier to configure.

Reply:
sure